server/datastore/mysql/migrations/tables/20230706141219_QueriesTableSchemaChanges.go

```diff
@@ -0,0 +1,49 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230706141219, Down_20230706141219)
+}
+
+func Up_20230706141219(tx *sql.Tx) error {
+	// If we want to drop the uniq constraint on queries.name, we first need to remove this
+	// constraint on scheduled_queries, due to a FK constraint scheduled_queries (query_name) =>
+	// queries (name).
```

Comment:
Sounds good. AFAICS we want to remove the `scheduled_queries` table (at the very end of the data migration).

Reply:
Ah, no, we can't delete the `scheduled_queries` table because it's used by 2017 packs.